docs/x509/verification.rst

```diff
@@ -17,15 +17,63 @@ the root of trust:
 
 .. code-block:: python
 
-    from cryptography.x509 import Certificate, DNSName, load_pem_x509_certificates
+    from cryptography.x509 import Certificate, DNSName, load_pem_x509_certificate, load_pem_x509_certificates
     from cryptography.x509.verification import PolicyBuilder, Store
     import certifi
 
+    from datetime import datetime
```

Comment:
Hmm, this makes it super verbose. Is there some way for us to make clear what perr and untrusted are without making the example 200 lines long

Reply:
If you're okay with it going back to being non-functional, I could probably do something like `peer, intermediates = load_pem_x509_certificates(open("whatever.pem").read())`. WDYT?

Reply:
Hmm, so using `doctest` + `testsetup` and then adding a comment explaining what they are would be my suggestion